pkg/cluster/config/types.go

```diff
@@ -38,6 +38,9 @@ type Cluster struct {
 
 	/* Advanced fields */
 
+	// Network contains cluster wide network settings
+	Network Network
```

Comment:
optionally this can be?
Network -> Networking

kubeadm:
https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#Networking

docker:
https://docs.docker.com/config/containers/container-networking/

Reply:
sure, no strong opinions on network vs networking here.

Reply:
amusingly s/Network/Networking broke finding the ports because it's in NetworkSettings

Reply:
done